core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -1036,6 +1036,48 @@ class GroupCoordinatorTest {
     assertEquals(Errors.NONE, syncGroupWithOldMemberIdResult.error)
   }
 
+ @Test
+ def staticMemberJoinStorageWithUnknownPersistenceFailure(): Unit = {
```

Comment:
nit: The test name is not so good here. How about `staticMemberRejoinWithUpdatedSessionAndRebalanceTimeoutsButCannotPersistChange`?

Reply:
fixed